website/docs/r/is_image.html.markdown

```diff
@@ -26,18 +26,40 @@ resource "ibm_is_image" "test_is_images" {
 }
 ```
 
+```terraform
+resource "ibm_is_image" "test_is_image2" {
+ name                   = "test_image"
+
+ //optional field, either of (href, operating_system) or source_volume is required
+ 
+ source_volume          = "xxxx-xxxx-xxxxxxx"
+ encrypted_data_key     = "eJxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx0="
+ encryption_key         = "crn:v1:bluemix:public:kms:us-south:a/6xxxxxxxxxxxxxxx:xxxxxxx-xxxx-xxxx-xxxxxxx:key:dxxxxxx-fxxx-4xxx-9xxx-7xxxxxxxx"
+   
+}
+```
+
+
 ## Argument Reference
 
 The following arguments are supported:
 
 * `name` - (Required, string) The descriptive name used to identify an image.
-* `href` - (Required, string) The path(SQL URL of COS Bucket Object) of an image to be uploaded.
-* `operating_system` - (Required, string) Description of underlying OS of an image.
+* `href` - (Optional, string) The path(SQL URL of COS Bucket Object) of an image to be uploaded.
+  * either `href` or `source_volume` is required
+* `operating_system` - (Optional, string) Description of underlying OS of an image.
+  * `operating_system` is required with `href`
 * `resource_group` - (Optional, Forces new resource, string) The resource group ID for this image.
 * `encrypted_data_key` - (Optional, Forces new resource, string) A base64-encoded, encrypted representation of the key that was used to encrypt the data for this image.
 * `encryption_key` - (Optional, Forces new resource, string) The CRN of the Key Protect Root Key or Hyper Protect Crypto Service Root Key for this resource.
 * `tags` - (Optional, array of strings) Tags associated with the image.
-
+* `source_volume` - (Optional, string) The volume id of the volume from which to create the image. The specified volume must:
+  * Originate from an image, which will be used to populate this image's operating system information.(boot type volumes)
```

Comment:
These statements (grammetically) you confirm from doc team because last pr from my doc team updated all vpc docs statements (some syntax and grammatical changes for sentences)